### ISSUE TITLE
Problem in intervalToDuration documentation

### ISSUE THREAD

**Issue by aminify** [date-fns/date-fns#3144]
There is an issue with the [intervalToDuration documentation](https://date-fns.org/v2.13.0/docs/intervalToDuration)

  ### The problem with intervalToDuration documentation

  The given example doesn't have a valid js syntax!
  ```
    const result = intervalToDuration({
      new Date(1929, 0, 15, 12, 0, 0),
      new Date(1968, 3, 4, 19, 5, 0)
    })
  ```

  ### Suggested edits

  The example should probably be converted to:
  ```
    const result = intervalToDuration({
      start: new Date(1929, 0, 15, 12, 0, 0),
      end: new Date(1968, 3, 4, 19, 5, 0)
    })
  ```

**Comment by aryans1319** [comment]
Can I work on this issue?

**Comment by WikiRik** [comment]
This has been resolved in the meantime;
https://github.com/date-fns/date-fns/blob/ddb34e0833f55020d90a1e6ccb682df3265337d6/src/intervalToDuration/index.ts#L27-L30

**Comment by fturmel** [comment]
Closing this. It was fixed 4 years ago f8ae348 and the issue is linking a stale v2.13 version of the docs.
